ingestion_server/ingestion_server/queries.py

```diff
@@ -33,6 +33,16 @@ def get_existence_queries(table):
     return exists_in_deleted_table, exists_in_mature_table
 
 
+def get_create_ext_query():
+    """
+    Get the query for creating the ``postgres_fdw`` extension, if it does not exist.
+
+    :return: the SQL query for creating the FDW extension
+    """
+
+    return "CREATE EXTENSION IF NOT EXISTS postgres_fdw;"
```

Comment:
Nit: Sorry, I hate to be a nuisance about this :sweat_smile: It seems like every other function in this module returns a `psycopg2.sql.SQL` object, perhaps this should too? On a personal note I think the string is totally fine, that just seems consistent with the rest of the code :woman_shrugging:

Reply:
I'm a sucker for consistency, and now that you've pointed it out, I can't unsee this. Updated in 5bac2d8.